packages/core/src/new-api/internal/reconciliation/futures/reconcileArtifactContractDeployment.ts

```diff
@@ -39,7 +39,9 @@ export function reconcileArtifactContractDeployment(
   if (!isEqual(future.value, executionState.value)) {
     return fail(
       future,
-      `Value has been changed from ${executionState.value} to ${future.value}`
+      `Value has been changed from ${executionState.value} to ${
```

Comment:
Should this compare the previous value from execution state with the proposed resolve parameter value?

Reply:
fixed